### ISSUE TITLE
Attributeerror: module tensorflow.contrib.lite.python.lite has no attribute TFLiteConverter

### ISSUE THREAD

**Issue by akotlarsky** [tensorflow/tensorflow#24277]
**System information**
- Raspberry Pi 3B+ Linux
- TensorFlow installed in Python using pip3; TensorFlow Lite installed and built (static library) from source
- TensorFlow version (installed in Python) is 1.11.0

Trying to convert a frozen graph file (.pb) to a TensorFlow Lite FlatBuffer file (.tflite), I am getting the error:
Attributeerror: module 'tensorflow.contrib.lite.python.lite' has no attribute 'TFLiteConverter'.

Here is the Python code (derived from https://www.tensorflow.org/lite/convert/python_api):

#!/usr/bin/python3
import tensorflow as tf
print("tf version = " + tf.__version__)
graph_def_file = "/home/pi/sols/demo/src/image_classification/network/fruit_models/frozen_graph.pb"
input_arrays = ["X"]
output_arrays = ["softmax"]
converter = tf.contrib.lite.TFLiteConverter.from_frozen_graph(
  graph_def_file, input_arrays, output_arrays)
tflite_model = converter.convert()
open("converted_model.tflite", "wb").write(tflite_model)

Please help!

**Comment by akotlarsky** [comment]
Created new issue:  https://github.com/tensorflow/tensorflow/issues/24323
